### ISSUE TITLE
Remove the deployment of firefox and thunderbird in home directory using policies

### ISSUE THREAD

**Issue by farfouille64** [clearlinux/distribution#1457]
As far as I understand, bundle of firefox copies the application in home directory because /usr is readonly and firefox binary packaged is mozilla genuine with auto update enable.
But using policy mechanism, it is possible and easy to turn off auto-update until the bundle is updated.
It is documented  here : https://support.mozilla.org/en-US/kb/customizing-firefox-using-policiesjson
In short, add the file `policies.json` in `distribution` directory of firefox wih the content :
```
{
 "policies": {
   "DisableAppUpdate": true
 }
}
```
And please, do the same thing for Thunderbird.

**Comment by bryteise** [comment]
We do some on the fly modifications to the upstream extracted content (/usr/bin/firefox is a shell script where you can see all of this). I think we have seen Mozilla change their stance on this but in the past they have been sticky about making changes to their official content. I would prefer, since Firefox is not built in Clear Linux today, that we instead host this as 3rd-party once that is better enabled.

**Comment by farfouille64** [comment]
I am not sure I understand all the subtleties of what change to their official content Mozilla accept, but the current default behavior of CL firefox script is source of error as it happened to me last friday (after Mozilla released 70.0.1 but before updated firefox CL bundle was pushed in the wild) :
1. launch firefox for the 1st time : CL firefox script copy version 70.0.0 in my home under `firefox` directory
2. firefox updates itself to version 70.0.1 (in my home) and update also my default firefox profile
3. quit firefox
4. remove the `firefox` directory (I thought I made a mistake)
5. relaunch firefox ; the CL script installs again firefox 70.0.0 and launches it
6. firefox refuses to use my default profile because it has already been updated (the profile, not firefox) to 70.0.1
I had to launch it with the option `--allow-downgrade`
I think that, at the minimum, the default directory in home to install firefox should be less visible than `firefox` in the root of home.
I know that the shell variable `FIREFOX_INSTALL_DIR` can be customized in `~/.config/firefox.conf` but still the default value is not very good and thunderbird has no `~/.config/thunderbird.conf`.

By 3rd-party do you mean provide firefox via flatpak ?

**Comment by bryteise** [comment]
> By 3rd-party do you mean provide firefox via flatpak ?

No, still via swupd but not coming from the default Clear Linux repository but an alternate 3rd-party repository that users would be able to add themselves.
